fixes.inc

```diff
@@ -205,6 +205,15 @@
  * AUTHOR:   im
  * AUTHOR:   Y_Less (https://github.com/Y-Less/)
  *
+ * FIX:      OnPlayerEnterVehicle_3
+ * PROBLEM:  No player animation when trying to enter the driver door of a locked vehicle
+ * SOLUTION: Leave the door unlocked and activate an animation when the player attemps
+ *           to enter a 'locked' vehicle.
+ * SEE:      "OnPlayerEnterVehicle".
+ * AUTHOR:   ziggi (https://github.com/ziggi)
+ * AUTHOR:   rt-2 (https://github.com/Y-Less/)
+ * POST:     https://github.com/ziggi/samp-enterfix
```

Comment:
Original post about this bug here: http://forum.sa-mp.com/showthread.php?t=560019